piapage/__init__.py

```
@@ -761,36 +754,36 @@ def get_is_color(self):
 
-            filtered_paragraphs_in_soup.append(paragraphs_in_soup[k])
-            filtered_paragraphs.append(paragraphs[k])
+        # loop thru the elements after "Description"
+        current_node = header_element.find_next_sibling()
+        while current_node is not None:
+            if isinstance(current_node, (Tag, NavigableString)):
+                caption_as_soup.append(current_node)
+
+            current_node = current_node.next_sibling
 
         background_indices = []
```

Ruff (0.15.2)

[error] Local variable `background_indices` is assigned to but never used

Remove assignment to unused variable `background_indices`

(F841)